app/routers/dogfinder.py

```diff
@@ -186,6 +197,7 @@ async def add_documents(documentRequest: DocumentRequest):
             
         # Embed the documents images
         embedding_results = embed_documents([create_pil_image(document.image) for document in documentRequest.documents], embedding_model)
+        request_dog_id = generate_dog_id()
```

Comment:
We assume here that all documents are the same dog id? why?

Reply:
The desired use case defined by @NirDiamant was to upload multiple images of the same dog at once, so we can tell they are all related to the same dog. So putting the same id to all of the images in the api achieves that. do you think the flow should work differently for stage 0?

Reply:
I want to be able to use this method to upload multiple dogs as well. Once we will have the DB we will be able to send the dog id in each object/document. But if it is a problem so sure lets do this change and later I'll figure out what to do with what I need.

Reply:
in the future when we have the sql db we can change the api to be related to a dog when uploading data instead, this is indeed a workaround for now